modules/nested-accordion/assets/scss/frontend.scss

```diff
@@ -89,6 +89,8 @@
 
 						& > svg {
 							fill: var(--n-accordion-icon-normal-color);
+						  	width: var(--n-accordion-icon-size);
+						  	height: auto;
```

Comment:
We are using a set `height` value in the span wrapper, and we are using `fit-content` for the `width`.

I am thinking could we follow that same pattern for the `svg` element?

It looks like here you have swopped the logic. Wouldn't it be easier to follow the pattern from the `span` parent?